tests/python/unittest/test_auto_scheduler_sketch_generation.py

```diff
@@ -93,10 +101,80 @@ def test_cpu_conv2d_winograd_sketch():
     '''
     assert len(sketches) == 3
 
+
+def test_cuda_matmul_sketch():
+    if not tvm.context("cuda", 0).exist:
+        return
+    sketches = generate_sketches(matmul_auto_scheduler_test, (512, 512, 512), 'cuda')
+    ''' 1 multi-level tiling sketch '''
```

Comment:
Do we need to do more fine-grained checks?
Only checking the number is not very robust.

Reply:
Now, we have more specific checks in this UT.